lein-finagle-clojure/src/leiningen/finagle_clojure.clj

```diff
@@ -33,15 +55,29 @@
     lein finagle-clojure scrooge :lint --help # shows available options for the linter
     lein finagle-clojure scrooge :lint -w # show linter warnings as well (warnings won't prevent compilation)"
   [project & options]
+  (doseq [f (classpath/get-classpath project)]
+    (pom/add-classpath f))
```

Comment:
It makes the calls to `io/resource` below able to resolve files on the classpath during the leiningen task.